src/screens/about/styles.tsx

```diff
@@ -0,0 +1,132 @@
+import { makeStyles } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+import { getMinMediaQuery } from '@styles';
+import { FormatSize } from '@material-ui/icons';
+
+export const useGetStyles = () => {
+  const theme: any = useTheme();
+  const useStyles = makeStyles({
+    root: {
+      background: theme.palette.custom.footer,
+      color: theme.palette.custom.onDarkSurface,
+      padding: '2rem 1rem',
+      display: 'flex',
+      flexDirection: 'column',
+      [getMinMediaQuery(theme.breakpoints.tablet)]: {
+        flexDirection: 'row',
+        alignItems: 'flex-start',
+      },
+    },
+    logoContainer: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'flex-end',
+      flexWrap: 'wrap',
+      color: theme.palette.custom.white,
+    },
+    mobileOnly: {
+      [getMinMediaQuery(theme.breakpoints.tablet)]: {
+        display: 'none',
+      },
+    },
+    tabletAndUp: {
+      display: 'none',
+      [getMinMediaQuery(theme.breakpoints.tablet)]: {
+        display: 'block',
+        borderRadius: '20px',
+      },
+    },
+    listContainer: {
+      [getMinMediaQuery(theme.breakpoints.tablet)]: {
+        flex: 1,
+        marginLeft: '2rem',
+        paddingTop: 0,
+      },
+    },
+    list: {
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    listItem: {
+      whiteSpace: 'nowrap',
+      width: 'auto',
+      padding: '0',
+      fontSize: theme.typography.body3.fontSize,
+      '& span': {
+        margin: '0 1rem',
+      },
+    },
+    playStore: {
```

Comment:
i'm pretty sure this is ripped directly off of the footer styles. You don't need this much

Reply:
i'll clean it up.